web/src/app/administration/components/classification-details/classification-details.component.ts

```diff
@@ -157,7 +165,8 @@ export class ClassificationDetailsComponent implements OnInit, OnDestroy {
 
   private async onSave() {
     this.requestInProgressService.setRequestInProgress(true);
-    if (this.action === ACTION.CREATE) {
+    if (this.action) {
+      this.classification.classificationId = null; // in case the id has been set, but a new classification should be created
```

Comment:
should this be done during save or creation? 
Personally I think that onCopy() should remove the Id.

Reply:
This way the reset button can use the id to reset the copy to the original and in my opinion it does not really matter when to id gets removed. It would be cleaner, but i liked the possibility to reset.

Reply:
Well.. the reset reset's the ID - I don't think this is desired..